core/tests/gauge-taiko automated tests/voiceover-tests/.vscode/extensions.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
All this stuff should not be here. Can it go in the root folder perhaps? Also what is it for?

Ditto for the next file.

Reply:
there is no need of both of the files!